releases/23.0.0.5/beta-instanton/Dockerfile.ubi.openjdk17

```diff
@@ -0,0 +1,101 @@
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-17-ea-jdk-ubi-amd64
```

Comment:
We must not have beta directories in the versioned release folders.

Reply:
I know. Your changes to InstantOn are so new and the automation script isn't updated to consider them.